StrangeIoC/scripts/strange/extensions/context/impl/CrossContext.cs

```diff
@@ -120,7 +121,7 @@ override public IContext AddContext(IContext context)
 		virtual public void AssignCrossContext(ICrossContextCapable childContext)
 		{
 			childContext.crossContextDispatcher = crossContextDispatcher;
-			childContext.injectionBinder.CrossContextBinder = injectionBinder.CrossContextBinder;
+			childContext.injectionBinder = new CrossContextInjectionBinder(injectionBinder.CrossContextBinder);
```

Comment:
I would need to test, but I think this has unintended consequences of removing local override bindings. e.g. I have a cross context value for a binding but my context wants to override it with a value.